hicommonwealth/commonwealth
# 7136 infiniteloopbug taketwo

## Link to Issue
Closes: #7136 

## Description of Changes
- Adds check in query to not exceed the limit (20) returns undefined otherwise
- Also added back the hook for resizing, that caused a regression and was not the fix

## "How We Fixed It"

- Added a check in the `getNextPageParam` to return undefined. Before it was just rapidly firing if the end was found. Which is not intended.

## Test Plan
- Go to discussions
- Open dev tools network tab
- On first arrival check to ensure the call isn't rapidly firing
- Do some resizing and ensure there's no regressions.

## Other
[First attempt](https://github.com/hicommonwealth/commonwealth/pull/7138)
